src/modules/task/task.service.ts

```diff
@@ -27,7 +27,7 @@ export class TaskService {
 
   async getOne(id: TaskModel['id']) {
     const task = await TaskModel.findOne({
-      where: { id },
+      where: { id, example: 'Просто пример' },
```

Comment:
```suggestion
      where: { id },
```

Это лишнее поле, ему здесь не место. Такого поля в базе нет -> фильтр работать не будет.